include/games.h

```diff
@@ -300,7 +300,7 @@ extern const unsigned char *joystick_type[];
 
 #ifdef __AQUARIUS__
 #ifdef DEFINE_JOYSTICK_TYPE
-	const unsigned char *joystick_type[] = {"QAOP-MN", "8246-05", "hjkl-sd", "Cursor"};
+	const unsigned char *joystick_type[] = {"Joystick 1", "Joystick 2", "QAOP-MN", "8246-05", "hjkl-sd", "Cursor"};
```

Comment:
This is just for the Plus?

Reply:
No. This save code should also work for the Aquarius.

Reply:
It uses the AY registers - I thought a regular Aquarius didn't have an AY?

Reply:
https://en.wikipedia.org/wiki/Mattel_Aquarius it had a single AY, here's the cheatsheet https://www.vdsteenoven.com/aquarius/hndcntrl.html

Reply:
Ah, the Mini-Expander add on.